ide/xsl/src/org/netbeans/modules/xsl/grammar/XSLGrammarQuery.java

```diff
@@ -378,7 +378,7 @@ private static Set getXslFunctions() {
 
     private static Set getXPathAxes() {
         if (xpathAxes == null) {
-            xpathAxes = new TreeSet(Arrays.asList(new String[]{"ancestor::", "ancestor-or-self::", // NOI18N
+            xpathAxes = new TreeSet<>(Arrays.<String>asList(new String[]{"ancestor::", "ancestor-or-self::", // NOI18N
```

Comment:
inferrable type informations
```suggestion
            xpathAxes = new TreeSet<>(Arrays.asList(new String[]{"ancestor::", "ancestor-or-self::", // NOI18N
```
strictly speaking the array creation is not needed as well

Reply:
changed..

Reply:
changed..